Algorithms/RunningSumArray.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Algorithms
+{
+    public class RunningSumArray
+    {
+        /// <summary>
+        /// Big O
+        /// Complejidad: O(n) - El arreglo solo se recorre una unica vez y se le resta la posición 
+        /// anterior que ya tiene el ultimo valor acumulado
+        /// </summary>
+        /// <param name="nums">Array de enteros</param>
+        /// <returns>Array de enteros</returns>
+        public int[] RunningSum(int[] nums)
+        {
+            int len = nums.Length;
+            for (int i = 0; i < len; i++)
+            {
+                if (i > 0)
+                    nums[i] = nums[i - 1] + nums[i];
+                else
```

Comment:
there's a way you can remove this if/else statement.

Reply:
Yes, There's a way. Actually first element of array never is affected with new value. Therefore loop for can start in 1.